core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -901,6 +901,7 @@ private[group] class GroupCoordinator(
                              generationId: Int,
                              offsetMetadata: immutable.Map[TopicIdPartition, OffsetAndMetadata],
                              responseCallback: immutable.Map[TopicIdPartition, Errors] => Unit,
+                             transactionalId: String,
```

Comment:
nit: Would it make sense to move `transactionalId` to right before `producerId`/`producerEpoch`?